reView/cli.py

```diff
@@ -129,3 +135,259 @@ def unpack_characterizations(
+                    'Default is to use state boundaries for CONUS from '
+                    'Natural Earth (1:50m scale), which is suitable for CONUS '
+                    'supply curves. For other region, it is recommended to '
+                    'provide a more appropriate boundaries dataset.'
+                    ))
+@click.option('--dpi', '-d', required=False,
+              default=600,
+              type=click.IntRange(min=0),
+              help='Dots-per-inch (DPI) for output images. Default is 600.')
+def make_maps(
+    supply_curve_csv, tech, out_folder, boundaries, dpi
+):
+    """
+    Generates standardized, presentation-quality maps for the input supply
+    curve, including maps for each of the following attributes:
+    Capacity (capacity), All-in LCOE (total_lcoe), Project LCOE (mean_lcoe),
+    LCOT (lcot), Capacity Density (derived column) [wind only]
+    """
+
+    out_path = Path(out_folder)
+    out_path.mkdir(exist_ok=True, parents=False)
+
+    supply_curve_df = pd.read_csv(supply_curve_csv)
+    supply_curve_gdf = gpd.GeoDataFrame(
+        supply_curve_df,
+        geometry=gpd.points_from_xy(
+            x=supply_curve_df['longitude'], y=supply_curve_df['latitude']
+        ),
+        crs="EPSG:4326"
+    )
+
+    boundaries_gdf = gpd.read_file(boundaries)
+    boundaries_singlepart_gdf = boundaries_gdf.explode(index_parts=True)
+
+    boundaries_dissolved = boundaries_gdf.unary_union
+    background_gdf = gpd.GeoDataFrame(
+        {"geometry": [boundaries_dissolved]},
+        crs=boundaries_gdf.crs
+    ).explode(index_parts=False)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=UserWarning)
+        map_extent = background_gdf.buffer(0.01).total_bounds
+
+    map_vars = {
+        "total_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "All-in LCOE ($/MWh)"
+        },
+        "mean_lcoe": {
+            "breaks": [25, 30, 35, 40, 45, 50, 60, 70],
+            "cmap": 'YlGn',
+            "legend_title": "Project LCOE ($/MWh)"
+        },
+        "lcot": {
+            "breaks": [5, 10, 15, 20, 25, 30, 35, 40, 50],
+            "cmap": 'YlGn',
+            "legend_title": "LCOT ($/MWh)",
+        }
+    }
+    if tech == "solar":
+        map_vars.update({
+            "capacity": {
+                "breaks": [100, 500, 1000, 2000, 3000, 4000],
+                "cmap": 'YlOrRd',
+                "legend_title": "Capacity (MW)"
+            }
+        })
+    elif tech == "wind":
+        supply_curve_gdf["capacity_density"] = (
+            supply_curve_gdf["capacity"] / supply_curve_gdf["area_sq_km"]
+        )
+        map_vars.update({
+            "capacity": {
+                "breaks": [100, 125, 150, 175, 200, 225],
+                "cmap": 'Blues',
+                "legend_title": "Capacity (MW)"
+            },
+            "capacity_density": {
```

Comment:
Why is wind the only tech that can plot capacity density? I assume there is some technical limitation here, but it seems like theoretically all technologies should be able to derive a capacity density